### DISCUSSION TITLE
Association setup response

### DISCUSSION THREAD

**Discussion by northmirko** [nextmn/.github#5] [Q&A] [answered]
Hi, does nextmn-upf implement config options for :
IE = 233,
IE = 43

![image](https://user-images.githubusercontent.com/2362694/202485011-c513d367-983b-469e-ab1d-80c7cfe2c06a.png)

Very nice code, by the way.

**Comment by louisroyer** [comment, marked as answer]
Hi,
IEs 233 and 43 are not implemented. PFPC support is a work in progress and does not even handle correctly every mandatory IEs.
For the moment, UE IP address pools have to be defined separately both in SMF and in UPF config files. In particular, `UEIP` feature is not supported : IP allocation has to be done entirely on SMF side.
This UPF has only been tested with Free5GC v3.2.0 SMF (but may work with newer versions). By curiosity, what SMF are you using?

**Comment by northmirko** [comment]
Hi, I am using PGW-C (Travelping erGW) LTE, not SMF/5G. I need to find some UPF (not necessarily fully functional) to complete setup for Sxb, at least to emulate Sxb, `cause at this moment I need to focus on Control Part of CUPS. Your implementation seem to be simple and extensible for testing and emulation of Sxb. Thank you for your response.
